### PR TITLE
Added comments to redirect users to using RobustaCLI to generate Helm…

### PR DESCRIPTION
**Pull request by pavangudiwada**
Added comments and links to the Installation page

### CODE REVIEW

File: helm/README.md [hunk 1]
```diff
@@ -9,4 +11,4 @@ If you want to upload a new chart version, follow these steps:
 4. From the directory, `helm`, run: `./upload_chart.sh`
 
 # Installing robusta with the helm chart
-See https://docs.robusta.dev/master/getting-started/installation.html
+See https://docs.robusta.dev/master/getting-started/installation.html.
```
Comment (aantn) [inline helm/README.md:14]: Get rid of the period. It can mess up the link.

File: helm/README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Robusta helm chart
-Robusta's helm chart is served from our gcp bucket
+Robusta's helm chart is served from our gcp bucket.
+
+NOTE: **Don't use the Helm values file directly**. Check the [Installation instructions](https://docs.robusta.dev/master/getting-started/installation.html) to generate one, using the RobustaCLI.
```
Comment (aantn) [inline helm/README.md:4]: `robusta cli` not `RobustaCLI`

File: helm/robusta/values.yaml [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # most of this file is documented at https://docs.robusta.dev/master/user-guide/configuration.html
 # if you have any questions, feel free to ask via github issues or Slack (see link on robusta.dev)
 
+# NOTE: Do not use this file to install Robusta. Check the Installation isntructions to generate your generated_values.yaml(Helm values file) using RobustaCLI.
```
Comment (aantn) [inline helm/robusta/values.yaml:4]: >  NOTE: Do not use this file to install Robusta.
Why not?

For the second half of the sentence, I would just say "see the installation instructions instead." No need to mention the generated_values or Robusta cli here IMO. They can read about it in the installation page.

Also, typo in `isntructions`.

Comment (pavangudiwada) [reply helm/robusta/values.yaml]: Fixed

File: helm/README.md (hunk 2 above)
Comment (pavangudiwada) [reply helm/README.md]: Fixed

File: helm/README.md (hunk 1 above)
Comment (pavangudiwada) [reply helm/README.md]: Removed.

File: helm/robusta/values.yaml [hunk 4]
```diff
@@ -1,6 +1,8 @@
 # most of this file is documented at https://docs.robusta.dev/master/user-guide/configuration.html
 # if you have any questions, feel free to ask via github issues or Slack (see link on robusta.dev)
 
+# NOTE: Do not use this file to install Robusta. See the installation instructions instead https://docs.robusta.dev/master/installation.html
```
Comment (aantn) [inline helm/robusta/values.yaml:4]: I would still explain why you shouldn't use this file directly. E.g. 

`Do not use this file as-is when installing Robusta. It has missing values that need to be filled in. See the installation...`

**Review by aantn — APPROVED** [review]

File: helm/robusta/values.yaml (hunk 4 above)
Comment (pavangudiwada) [reply helm/robusta/values.yaml]: Thank you, fixed it.

**Pull request merged**